packages/sdk/src/FuelWalletConnection.ts

```diff
@@ -16,6 +17,8 @@ import { WindowConnection } from './connections/WindowConnection';
 import { getTransactionSigner } from './utils/getTransactionSigner';
 
 export class FuelWalletConnection extends WindowConnection {
+  readonly events = FuelEventTypes;
+
```

Comment:
any specific reason for adding it here inside class?

we could always use from `FuelEventTypes` for example, no needs to `this.events.NETWORK` ..